eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -429,6 +429,12 @@ object LightningMessageCodecs {
 
   //
 
+  val recommendedFeeratesCodec: Codec[RecommendedFeerates] = (
+    ("chainHash" | blockHash) ::
+      ("fundingFeerate" | feeratePerKw) ::
+      ("commitmentFeerate" | feeratePerKw) ::
```

Comment:
Might as well put the `closingFeerate`?
```suggestion
      ("commitmentFeerate" | feeratePerKw) ::
      ("closingFeerate" | feeratePerKw) ::
```